README.md

```diff
@@ -27,36 +26,65 @@ With "read the manual" fatigue and lengthy selection processes wearing on us eve
 
-    - auth-api: REST service for generating auth tokens
-    - auth-table-based-service: Service to store tokens in the database
-    - authentication: Interface for working with auth providers
+- [persistence-api](persistence-api): Interface for working with persistence services.  as   
+- [persistence-common](persistence-common): Utilities shared by the persistence services
+- [persistence-cassandra-3.11](persistence-cassandra-3.11): Joins C* 3.11 cluster as coordinator-only node (does not store data),
+mocks C* system tables for native driver integration,
+executes requests with C* storage nodes using C* QueryHandler/QueryProcessor,
+converts internal C* objects and ResultSets to Stargate Datastore objects.
+- [persistence-cassandra-4.0](persistence-cassandra-4.0): (same as above but for Cassandra 4.0)
+- [persistence-dse-6.8](persistence-dse-6.8): (same as above but for DataStax Enterprise 6.8)
 
-![image](assets/stargate-modules-preview-version.png#center)
+#### Authentication Services
+Stargate coordinator nodes also support a pluggable authentication approach.
+
+- [authnz](authnz): Interface for working with auth providers
+- [auth-api](auth-api): REST service for generating auth tokens
+- [auth-table-based-service](auth-table-based-service): Service to store tokens in the database
+- [auth-jtw-based-service](auth-jwt-based-service): Service to authenticate using externally generated JSON Web Tokens (JWTs)
+
+#### Coordinator Node Testing
+The following modules provide support for testing:
+
+- [testing](testing): Integration test suite for the coordinator node modules
+- [persistence-test](persistence-test): Common utilities for testing persistence services
+
+Instructions for running and extending the test suite can be found in the [developer guide](DEV_GUIDE.md).
 
 ## Repositories
 
-- [stargate/stargate](https://github.com/stargate/stargate): This repository is the primary entry point to the project. It contains all of the modules.
-- [stargate/docker-images](https://github.com/stargate/docker-images): This repository contains the Dockerfiles used to create and publish images to https://hub.docker.com/orgs/stargateio
+Here is an overview of the key repositories in the Stargate GitHub organization:
+
+- [stargate/stargate](https://github.com/stargate/stargate): This repository is the primary entry point to the project. It is a monorepo containing all of the Stargate modules.
 - [stargate/docs](https://github.com/stargate/docs): This repository contains the user docs hosted on [stargate.io](https://stargate.io)
 - [stargate/website](https://github.com/stargate/website): This repository contains the code for the website hosted on [stargate.io](https://stargate.io)
```

Comment:
I think it would be useful to add stargate/docker-images, and maybe the various grpc client repos.

Reply:
@polandll `docker-images` will go away as it is only used by Stargate V1. I think.

Reply:
Tatu is correct about `docker-images`. I can add a general statement about gRPC repos rather than an exhaustive list.